### PR TITLE
AoA Shadows

### PR DESCRIPTION
**Pull request by SkyJedi**

### CODE REVIEW

File: src/card-scripts/aoa/shadows/SackOfCoins.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { GameState } from "../../../shared/gamestate/GameState"
+import { activePlayerState, allCreatures, dealDamage } from "../../ScriptUtils"
+import { Creature } from "../../../shared/gamestate/Creature"
+
+const cardScript: CardScript = {
+    // Play: Deal 1D to a creature for each A in your pool.
+    amber: () => 1,
+    onPlay: {
+        validTargets: allCreatures,
+        numberOfTargets: () => 1,
```
Comment (Dunkoro) [inline src/card-scripts/aoa/shadows/SackOfCoins.ts]: Number of targets varies, not amount of damage.

File: src/card-scripts/aoa/shadows/ScowlyCaper.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { cardScripts } from "../../CardScripts"
+import { GameState } from "../../../shared/gamestate/GameState"
+import { destroyCards, getNeighbors, inactivePlayerState, moveCreature } from "../../ScriptUtils"
+import { Creature } from "../../../shared/gamestate/Creature"
+
+const cardScript: CardScript = {
+    // Skirmish. Scowly Caper enters play under your opponent’s control and can be used as if it belonged to any house.
+    // At the end of your turn, destroy one of Scowly Caper’s neighbors.
+    power: () => 2,
+    skirmish: () => true,
```
Comment (Dunkoro) [inline src/card-scripts/aoa/shadows/ScowlyCaper.ts]: canAlwaysUse => true

File: src/card-scripts/aoa/shadows/Murkens.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { shuffle } from "lodash"
+
+const cardScript: CardScript = {
+    // Play: Choose a random card in your opponent’s archives or the top card of your opponent’s deck. Play that card as if it were yours.
+    power: () => 2,
+    onPlay: {
+        selectFromChoices: () => ["Enemy Archives", "Enemy Deck"],
+        perform: (state: GameState, config: CardActionConfig) => {
+            const target = config.selection === "Enemy Archive" ? shuffle(inactivePlayerState(state).archives) : inactivePlayerState(state).library
+            if (target.length > 0) {
```
[the remark this answers is not in the document]
Comment (SkyJedi) [reply src/card-scripts/aoa/shadows/Murkens.ts]: shuffle()

**Comment by SkyJedi** [comment]
comments responded to and conflicts all fixed up

**Pull request merged**